### PR TITLE
Add radium block provider to RPC endpoints page

### PR DESCRIPTION
**Pull request by kapetan3sid**
### Description

Please explain the changes this PR addresses here.

### Checklist

- [x] I have added a label to this PR 🏷️
- [x] I have run my changes through Grammarly
- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

**Review by themacexpert — CHANGES_REQUESTED** [review]
Is RadiumBlock a public endpoint? I get these errors whenever I try to query a method like getBalance or estimateGas: 

`<html>

<head>
    <title>403 Forbidden</title>
</head>

<body>
    <center>
        <h1>403 Forbidden</h1>
    </center>
    <hr>
    <center>openresty/1.25.3.2</center>
</body>

</html>`

File: .snippets/text/builders/get-started/endpoints/moonbeam.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
 === "HTTPS"
     |   Provider   |                                            RPC URL                                            |   Limits    |
     |:------------:|:---------------------------------------------------------------------------------------------:|:-----------:|
-    |     1RPC     |                             <pre>```https://1rpc.io/glmr```</pre>                             | Unspecified |
     |    Blast     |                     <pre>```https://moonbeam.public.blastapi.io```</pre>                      | 80 req/sec  |
     |   Dwellir    |                       <pre>```https://moonbeam-rpc.dwellir.com```</pre>                       | 20 req/sec  |
     |  OnFinality  |                  <pre>```https://moonbeam.api.onfinality.io/public```</pre>                   | 40 req/sec  |
     |  UnitedBloc  |                       <pre>```https://moonbeam.unitedbloc.com```</pre>                        | 32 req/sec  |
+     |  RadiumBlock  |                       <pre>```https://moonbeam.public.curie.radiumblock.co```</pre>                        | 200 req/sec   |
```
Comment (themacexpert) [inline .snippets/text/builders/get-started/endpoints/moonbeam.md:8]:
```suggestion
     |  RadiumBlock  |                       <pre>```https://moonbeam.public.curie.radiumblock.co/http```</pre>                        | 200 req/sec   |
```

File: .snippets/text/builders/get-started/endpoints/moonriver.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
       |   Dwellir    |                       <pre>```https://moonriver-rpc.dwellir.com```</pre>                       | 20 req/sec |
       |  OnFinality  |                  <pre>```https://moonriver.api.onfinality.io/public```</pre>                   | 40 req/sec |
       |  UnitedBloc  |                       <pre>```https://moonriver.unitedbloc.com```</pre>                        | 32 req/sec |
+      |  RadiumBlock  |                       <pre>```https://moonriver.public.curie.radiumblock.co```</pre>                        | 200 req/sec |
```
Comment (themacexpert) [inline .snippets/text/builders/get-started/endpoints/moonriver.md:8]:
```suggestion
      |  RadiumBlock  |                       <pre>```https://moonriver.public.curie.radiumblock.co/http```</pre>                        | 200 req/sec |
```

File: .snippets/text/builders/get-started/endpoints/moonriver.md [hunk 3]
```diff
@@ -13,3 +14,4 @@
     |  Dwellir   |       <pre>```wss://moonriver-rpc.dwellir.com```</pre>       | 20 req/sec |
     | OnFinality | <pre>```wss://moonriver.api.onfinality.io/public-ws```</pre> | 40 req/sec |
     | UnitedBloc |       <pre>```wss://moonriver.unitedbloc.com```</pre>        | 32 req/sec |
+      | RadiumBlock |<pre>```wss://moonriver.public.curie.radiumblock.co```</pre>        | 200 req/sec |
```
Comment (themacexpert) [inline .snippets/text/builders/get-started/endpoints/moonriver.md:17]:
```suggestion
      | RadiumBlock |<pre>```wss://moonriver.public.curie.radiumblock.co/ws```</pre>        | 200 req/sec |
```

File: .snippets/text/builders/get-started/endpoints/moonbeam.md [hunk 4]
```diff
@@ -15,3 +17,5 @@
     |  Dwellir   |       <pre>```wss://moonbeam-rpc.dwellir.com```</pre>       | 20 req/sec |
     | OnFinality | <pre>```wss://moonbeam.api.onfinality.io/public-ws```</pre> | 40 req/sec |
     | UnitedBloc |       <pre>```wss://moonbeam.unitedbloc.com```</pre>        | 32 req/sec |
+    | RadiumBlock |       <pre>```wss://moonbeam.public.curie.radiumblock.co```</pre>       | 200 req/sec |
```
Comment (themacexpert) [inline .snippets/text/builders/get-started/endpoints/moonbeam.md:20]:
```suggestion
    | RadiumBlock |       <pre>```wss://moonbeam.public.curie.radiumblock.co/ws```</pre>       | 200 req/sec |
```

**Review by themacexpert — CHANGES_REQUESTED** [review]
Added changes

**Comment by eshaben** [comment]
Drive by comment: I'd love to see the tables formatted after the suggestions have been added 🙂 🚗

**Review by themacexpert — APPROVED** [review]
LGTM

**Pull request merged**